test/e2e/traceflow_test.go

```diff
@@ -1359,7 +1350,10 @@ func testTraceflowInterNode(t *testing.T, data *TestData) {
 			},
 		},
 		{
-			name:      "serviceTraceflowIPv4",
+			name: "serviceTraceflowIPv4",
+			skipIfNeeded: func(t *testing.T) {
+				skipIfProxyDisabled(t)
```

Comment:
out of curiosity, why this can run with proxy disabled previously? or it never ran because VMC testbeds always enable proxy?

Reply:
* Jenkins testbeds always have AntreaProxy enabled
* The inter-Node Traceflow tests were disabled for Kind previously because of the tunnel packet drop issue, and the only CI job we have with AntreaProxy disabled is on Kind